task/ansible-builder/0.1/ansible-builder.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: ansible-builder
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.21.0"
```

Comment:
Any specific reason for keeping the min pipelines version as 0.21.0 and not 0.12.0?

Reply:
Not really, I only tested this with 0.21 version..thats why. If you think there wont be any issues, I can change that to 0.12.0 obs.